crates/matrix-sdk/tests/integration/encryption/backups.rs

```diff
@@ -70,6 +76,20 @@ async fn mount_once(
         .await;
 }
 
+async fn just_mount(
```

Comment:
Instead of this function, I'd be fine immediately duplicating this function's body at all the call sites, so we immediately see what's being mocked and how.

Reply:
done